vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -73,10 +73,17 @@
  * <p>
  * To avoid manually dealing with responsive breakpoints, Form Layout provides
  * an auto-responsive mode that automatically creates and adjusts fixed-width
- * columns based on the container's available space. The
- * {@link #setColumnWidth(String) columnWidth} and {@link #setMaxColumns(int)
- * maxColumns} properties control the column width (13em by default) and the
- * maximum number of columns (10 by default) that the Form Layout can create.
+ * columns based on the container's available space.
+ * <p>
+ * The property
+ * <ul>
+ * <li>{@link #setColumnWidth(String) columnWidth} controls the column width
+ * (13em by default).</li>
+ * <li>{@link #setMaxColumns(int) maxColumns} controls the maximum number of
+ * columns (10 by default) that the Form Layout can create.</li>
+ * <li>{@link #setMinColumns(int) minColumns} controls the minimum number of
+ * columns (1 by default) that the Form Layout will create.</li>
+ * </ul>
```

Comment:
Maybe this could be improved a bit to move the default values to the end of each sentence. 
```suggestion
 * To control the number of columns and their widths, you can use the following
 * properties:
 * <ul>
 * <li>{@link #setColumnWidth(String) columnWidth} - controls the column width
 * (13em by default).</li>
 * <li>{@link #setMaxColumns(int) maxColumns} - controls the maximum number of
 * columns that the layout can create (10 by default).</li>
 * <li>{@link #setMinColumns(int) minColumns} - controls the minimum number of
 * columns that the layout will create (1 by default).</li>
 * </ul>
```